src/factories/sm64/AnimationFactory.cpp

```diff
@@ -1,5 +1,8 @@
 #include "AnimationFactory.h"
 #include "spdlog/spdlog.h"
+#include <Decompressor.h>
+
+#define ANIMINDEX_COUNT(boneCount) (sizeof(boneCount) + 1) * 6 * sizeof(uint16_t)
```

Comment:
```suggestion
#define ANIMINDEX_COUNT(boneCount) (sizeof(boneCount) + 1) * 6 / sizeof(uint16_t)
```
i think this should be a divide

Reply:
nvm im wrong